src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -16,8 +16,7 @@
     <Compile Include="ReadOnlySpan\Contains.T.cs" />
     <Compile Include="Span\Reflection.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="AllocationHelper.cs" />
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
```

Comment:
Move this to the above existing ItemGroup? `<ItemGroup Condition="'$(IncludePartialFacadeTests)' == 'true'">`

Reply:
I wasn't sure we wanted to build this for UAP. We don't really have a new version number for UAP I think? Want to understand the versioning here and asked about it above.